src/components/SharedComponents/DeletePhotoDialog.js

```diff
@@ -18,28 +18,44 @@ type Props = {
   photoUriToDelete: ?string,
   photoUris: Array<string>,
   setPhotoUris: Function,
-  hideDialog: Function
+  hideDialog: Function,
+  evidenceToAdd?: Array<string>,
+  setEvidenceToAdd?: Function
 }
 
 const DeletePhotoDialog = ( {
   deleteDialogVisible,
   photoUriToDelete,
   photoUris,
   setPhotoUris,
-  hideDialog
+  hideDialog,
+  evidenceToAdd,
+  setEvidenceToAdd
```

Comment:
This is the second level at which many of these props get passed down. That's good for modularity, but isn't that kind of the problem context/providers are supposed to solve?